2018-05-15T17:32:11.139Z - error: Error when processing OData PATCH: /odata/data(Qgif1ozghDsk3qMR) Error: EPERM: operation not permitted, open 'C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\data\fs.lock'
2018-05-15T18:06:40.190Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Tue May 15 2018 20:06:39 GMT+0200 (Paris, Madrid (heure d’été)), pid=8184, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=44703744, heapTotal=33697792, heapUsed=22680692, external=291040, loadavg=[0, 0, 0], uptime=8061.8100569, trace=[column=11, file=util.js, function=Object._errnoException, line=1022, method=_errnoException, native=false, column=20, file=util.js, function=_exceptionWithHostPort, line=1044, method=null, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1198, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object._errnoException (util.js:1022:11),     at _exceptionWithHostPort (util.js:1044:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)]
2018-05-15T18:06:40.200Z - error: uncaughtException: socket hang up date=Tue May 15 2018 20:06:40 GMT+0200 (Paris, Madrid (heure d’été)), pid=8184, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=46002176, heapTotal=34746368, heapUsed=15515920, external=41412, loadavg=[0, 0, 0], uptime=8061.8230577, trace=[column=15, file=_http_client.js, function=createHangUpError, line=331, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=363, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=121, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=211, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=567, method=close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:331:15),     at Socket.socketCloseListener (_http_client.js:363:23),     at emitOne (events.js:121:20),     at Socket.emit (events.js:211:7),     at TCP._handle.close [as _onclose] (net.js:567:12)]
2018-05-15T18:07:26.460Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:80 date=Tue May 15 2018 20:07:26 GMT+0200 (Paris, Madrid (heure d’été)), pid=6776, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=45043712, heapTotal=33697792, heapUsed=22179636, external=291040, loadavg=[0, 0, 0], uptime=8108.0807034, trace=[column=11, file=util.js, function=Object._errnoException, line=1022, method=_errnoException, native=false, column=20, file=util.js, function=_exceptionWithHostPort, line=1044, method=null, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1198, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:80,     at Object._errnoException (util.js:1022:11),     at _exceptionWithHostPort (util.js:1044:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)]
2018-05-15T18:07:26.468Z - error: uncaughtException: socket hang up date=Tue May 15 2018 20:07:26 GMT+0200 (Paris, Madrid (heure d’été)), pid=6776, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=45375488, heapTotal=33697792, heapUsed=22241140, external=291064, loadavg=[0, 0, 0], uptime=8108.0917041, trace=[column=15, file=_http_client.js, function=createHangUpError, line=331, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=363, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=121, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=211, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=567, method=close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:331:15),     at Socket.socketCloseListener (_http_client.js:363:23),     at emitOne (events.js:121:20),     at Socket.emit (events.js:211:7),     at TCP._handle.close [as _onclose] (net.js:567:12)]
2018-05-15T18:15:28.813Z - error: Error during processing request at http://localhost:5488/api/report, details: Not initialized, you need to call jsreport.init().then before rendering, stack: Error: Not initialized, you need to call jsreport.init().then before rendering
    at Reporter.render (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-core\lib\reporter.js:246:13)
    at Object.reporter.express.render (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\lib\routes.js:44:14)
    at app.post (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\lib\routes.js:74:22)
    at Layer.handle [as handle_request] (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:275:10)
    at app.use (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\lib\routes.js:11:5)
    at Layer.handle [as handle_request] (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\node_modules\jsreport-express\node_modules\express\lib\router\index.js:284:7
2018-05-15T19:18:33.491Z - error: uncaughtException: connect ECONNREFUSED 127.0.0.1:5488 date=Tue May 15 2018 21:18:33 GMT+0200 (Paris, Madrid (heure d’été)), pid=4836, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=65998848, heapTotal=52240384, heapUsed=38745168, external=1828200, loadavg=[0, 0, 0], uptime=368.1120361, trace=[column=11, file=util.js, function=Object._errnoException, line=1022, method=_errnoException, native=false, column=20, file=util.js, function=_exceptionWithHostPort, line=1044, method=null, native=false, column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1198, method=afterConnect [as oncomplete], native=false], stack=[Error: connect ECONNREFUSED 127.0.0.1:5488,     at Object._errnoException (util.js:1022:11),     at _exceptionWithHostPort (util.js:1044:20),     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)]
2018-05-15T19:18:33.701Z - error: uncaughtException: socket hang up date=Tue May 15 2018 21:18:33 GMT+0200 (Paris, Madrid (heure d’été)), pid=4836, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=67821568, heapTotal=53059584, heapUsed=39910748, external=2306415, loadavg=[0, 0, 0], uptime=368.3240482, trace=[column=15, file=_http_client.js, function=createHangUpError, line=331, method=null, native=false, column=23, file=_http_client.js, function=Socket.socketCloseListener, line=363, method=socketCloseListener, native=false, column=20, file=events.js, function=emitOne, line=121, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=211, method=emit, native=false, column=12, file=net.js, function=TCP._handle.close [as _onclose], line=567, method=close [as _onclose], native=false], stack=[Error: socket hang up,     at createHangUpError (_http_client.js:331:15),     at Socket.socketCloseListener (_http_client.js:363:23),     at emitOne (events.js:121:20),     at Socket.emit (events.js:211:7),     at TCP._handle.close [as _onclose] (net.js:567:12)]
2018-05-15T19:51:52.643Z - error: uncaughtException: SQLITE_ERROR: table entreeStock has no column named gainMarge date=Tue May 15 2018 21:51:52 GMT+0200 (Paris, Madrid (heure d’été)), pid=6536, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55046144, heapTotal=36577280, heapUsed=31664348, external=255329, loadavg=[0, 0, 0], uptime=2367.2663812, trace=[], stack=[Error: SQLITE_ERROR: table entreeStock has no column named gainMarge]
2018-05-15T19:55:41.794Z - error: uncaughtException: SQLITE_ERROR: table entreeStock has no column named gainMarge date=Tue May 15 2018 21:55:41 GMT+0200 (Paris, Madrid (heure d’été)), pid=7940, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74305536, heapTotal=55156736, heapUsed=40577048, external=2278675, loadavg=[0, 0, 0], uptime=2596.4164878, trace=[], stack=[Error: SQLITE_ERROR: table entreeStock has no column named gainMarge]
2018-05-15T19:56:07.332Z - error: uncaughtException: SQLITE_ERROR: table entreeStock has no column named gainMarge date=Tue May 15 2018 21:56:07 GMT+0200 (Paris, Madrid (heure d’été)), pid=6196, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73355264, heapTotal=54632448, heapUsed=41185600, external=2380661, loadavg=[0, 0, 0], uptime=2621.9559486, trace=[], stack=[Error: SQLITE_ERROR: table entreeStock has no column named gainMarge]
2018-05-15T20:27:07.959Z - error: uncaughtException: SQLITE_ERROR: table entreeStock has no column named gainMarge date=Tue May 15 2018 22:27:07 GMT+0200 (Paris, Madrid (heure d’été)), pid=7792, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55128064, heapTotal=36577280, heapUsed=31892068, external=279999, loadavg=[0, 0, 0], uptime=4482.5813703, trace=[], stack=[Error: SQLITE_ERROR: table entreeStock has no column named gainMarge]
2018-05-16T00:06:09.891Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:06:09 GMT+0200 (Paris, Madrid (heure d’été)), pid=3516, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=58134528, heapTotal=37625856, heapUsed=32497320, external=223869, loadavg=[0, 0, 0], uptime=17624.5150459, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:07:16.250Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:07:16 GMT+0200 (Paris, Madrid (heure d’été)), pid=5532, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74211328, heapTotal=55681024, heapUsed=40487816, external=2310294, loadavg=[0, 0, 0], uptime=17690.8738414, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:08:04.982Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:08:04 GMT+0200 (Paris, Madrid (heure d’été)), pid=6876, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=58163200, heapTotal=39428096, heapUsed=34769040, external=438139, loadavg=[0, 0, 0], uptime=17739.6046286, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:09:42.013Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:09:41 GMT+0200 (Paris, Madrid (heure d’été)), pid=2532, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55750656, heapTotal=37625856, heapUsed=32208520, external=266524, loadavg=[0, 0, 0], uptime=17836.6341784, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:10:05.790Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:10:05 GMT+0200 (Paris, Madrid (heure d’été)), pid=7768, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=72826880, heapTotal=55156736, heapUsed=35130172, external=466632, loadavg=[0, 0, 0], uptime=17860.4135385, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:11:31.820Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:11:31 GMT+0200 (Paris, Madrid (heure d’été)), pid=7176, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=56168448, heapTotal=37625856, heapUsed=32345388, external=368311, loadavg=[0, 0, 0], uptime=17946.441459, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:15:00.072Z - error: uncaughtException: SQLITE_ERROR: no such table: achat date=Wed May 16 2018 02:15:00 GMT+0200 (Paris, Madrid (heure d’été)), pid=4304, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=56037376, heapTotal=37101568, heapUsed=31957500, external=241078, loadavg=[0, 0, 0], uptime=18154.6953704, trace=[], stack=[Error: SQLITE_ERROR: no such table: achat]
2018-05-16T00:17:36.192Z - error: uncaughtException: SQLITE_ERROR: 12 values for 11 columns date=Wed May 16 2018 02:17:36 GMT+0200 (Paris, Madrid (heure d’été)), pid=6984, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55549952, heapTotal=37101568, heapUsed=31953208, external=310958, loadavg=[0, 0, 0], uptime=18310.8153, trace=[], stack=[Error: SQLITE_ERROR: 12 values for 11 columns]
2018-05-16T00:18:20.560Z - error: uncaughtException: SQLITE_ERROR: 12 values for 11 columns date=Wed May 16 2018 02:18:20 GMT+0200 (Paris, Madrid (heure d’été)), pid=7800, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=54341632, heapTotal=36577280, heapUsed=31756248, external=286336, loadavg=[0, 0, 0], uptime=18355.1828377, trace=[], stack=[Error: SQLITE_ERROR: 12 values for 11 columns]
2018-05-16T00:21:41.224Z - error: uncaughtException: SQLITE_ERROR: 12 values for 11 columns date=Wed May 16 2018 02:21:41 GMT+0200 (Paris, Madrid (heure d’été)), pid=7900, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55705600, heapTotal=37625856, heapUsed=32294692, external=254311, loadavg=[0, 0, 0], uptime=18555.847315, trace=[], stack=[Error: SQLITE_ERROR: 12 values for 11 columns]
2018-05-16T00:22:39.077Z - error: uncaughtException: SQLITE_ERROR: 12 values for 11 columns date=Wed May 16 2018 02:22:39 GMT+0200 (Paris, Madrid (heure d’été)), pid=5580, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74072064, heapTotal=54632448, heapUsed=40436888, external=2454625, loadavg=[0, 0, 0], uptime=18613.699624, trace=[], stack=[Error: SQLITE_ERROR: 12 values for 11 columns]
2018-05-16T00:23:47.305Z - error: uncaughtException: SQLITE_ERROR: 12 values for 11 columns date=Wed May 16 2018 02:23:47 GMT+0200 (Paris, Madrid (heure d’été)), pid=6748, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=55427072, heapTotal=37101568, heapUsed=32352984, external=336399, loadavg=[0, 0, 0], uptime=18681.9285265, trace=[], stack=[Error: SQLITE_ERROR: 12 values for 11 columns]
2018-05-18T22:01:27.851Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:01:27 GMT+0200 (Paris, Madrid (heure d’été)), pid=3120, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73347072, heapTotal=55156736, heapUsed=35406156, external=452623, loadavg=[0, 0, 0], uptime=2907.4731971, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:01:57.199Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:01:57 GMT+0200 (Paris, Madrid (heure d’été)), pid=7860, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73105408, heapTotal=55156736, heapUsed=34940144, external=427159, loadavg=[0, 0, 0], uptime=2936.8218758, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:03:34.015Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:03:33 GMT+0200 (Paris, Madrid (heure d’été)), pid=888, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74637312, heapTotal=54632448, heapUsed=37749944, external=2165456, loadavg=[0, 0, 0], uptime=3033.6374133, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:04:57.610Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:04:57 GMT+0200 (Paris, Madrid (heure d’été)), pid=4940, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=54353920, heapTotal=36577280, heapUsed=31638300, external=255060, loadavg=[0, 0, 0], uptime=3117.2321947, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:06:14.680Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:06:14 GMT+0200 (Paris, Madrid (heure d’été)), pid=4220, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74207232, heapTotal=54632448, heapUsed=40919356, external=2479574, loadavg=[0, 0, 0], uptime=3194.3026028, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:09:19.548Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:09:19 GMT+0200 (Paris, Madrid (heure d’été)), pid=7028, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74420224, heapTotal=54738944, heapUsed=38951508, external=2089064, loadavg=[0, 0, 0], uptime=3379.1691766, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:11:03.770Z - error: uncaughtException: SQLITE_ERROR: no such table: Utilisateur date=Sat May 19 2018 00:11:03 GMT+0200 (Paris, Madrid (heure d’été)), pid=7856, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73805824, heapTotal=54108160, heapUsed=36007612, external=2038136, loadavg=[0, 0, 0], uptime=3483.3911377, trace=[], stack=[Error: SQLITE_ERROR: no such table: Utilisateur]
2018-05-18T22:13:13.718Z - error: uncaughtException: SQLITE_ERROR: no such table: utilisateur date=Sat May 19 2018 00:13:13 GMT+0200 (Paris, Madrid (heure d’été)), pid=4528, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=68956160, heapTotal=51486720, heapUsed=32011304, external=510960, loadavg=[0, 0, 0], uptime=3613.3385703, trace=[], stack=[Error: SQLITE_ERROR: no such table: utilisateur]
2018-05-18T22:13:42.006Z - error: uncaughtException: SQLITE_ERROR: no such table: utilisateur date=Sat May 19 2018 00:13:41 GMT+0200 (Paris, Madrid (heure d’été)), pid=7880, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=75116544, heapTotal=55263232, heapUsed=37410576, external=2097256, loadavg=[0, 0, 0], uptime=3641.6271883, trace=[], stack=[Error: SQLITE_ERROR: no such table: utilisateur]
2018-05-18T22:16:06.731Z - error: uncaughtException: SQLITE_ERROR: no such table: utilisateur date=Sat May 19 2018 00:16:06 GMT+0200 (Paris, Madrid (heure d’été)), pid=7232, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=74887168, heapTotal=55263232, heapUsed=38432336, external=2148184, loadavg=[0, 0, 0], uptime=3786.3544663, trace=[], stack=[Error: SQLITE_ERROR: no such table: utilisateur]
2018-05-18T22:20:39.677Z - error: uncaughtException: SQLITE_ERROR: no such table: utilisateur date=Sat May 19 2018 00:20:39 GMT+0200 (Paris, Madrid (heure d’été)), pid=7072, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73650176, heapTotal=55156736, heapUsed=41439932, external=2312902, loadavg=[0, 0, 0], uptime=4059.2990778, trace=[], stack=[Error: SQLITE_ERROR: no such table: utilisateur]
2018-05-19T02:44:38.193Z - error: uncaughtException: SQLITE_ERROR: near "categoryId": syntax error date=Sat May 19 2018 04:44:37 GMT+0200 (Paris, Madrid (heure d’été)), pid=3788, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=72208384, heapTotal=41295872, heapUsed=35710532, external=219098, loadavg=[0, 0, 0], uptime=18764.7337108, trace=[], stack=[Error: SQLITE_ERROR: near "categoryId": syntax error]
2018-05-19T02:49:08.202Z - error: uncaughtException: SQLITE_ERROR: near "categoryId": syntax error date=Sat May 19 2018 04:49:08 GMT+0200 (Paris, Madrid (heure d’été)), pid=7736, uid=null, gid=null, cwd=C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\NodeJS-REST-API-SQLite-master\stock_api\index.js], rss=73789440, heapTotal=55681024, heapUsed=34771148, external=362610, loadavg=[0, 0, 0], uptime=19034.7441545, trace=[], stack=[Error: SQLITE_ERROR: near "categoryId": syntax error]
2018-05-19T14:03:57.150Z - error: uncaughtException: SQLITE_ERROR: near "categoryId": syntax error date=Sat May 19 2018 16:03:57 GMT+0200 (Paris, Madrid (heure d’été)), pid=6052, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=71442432, heapTotal=39129088, heapUsed=34655080, external=244088, loadavg=[0, 0, 0], uptime=1700.7710397, trace=[], stack=[Error: SQLITE_ERROR: near "categoryId": syntax error]
2018-05-19T14:09:51.121Z - error: uncaughtException: SQLITE_ERROR: near "categoryId": syntax error date=Sat May 19 2018 16:09:51 GMT+0200 (Paris, Madrid (heure d’été)), pid=6532, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=54026240, heapTotal=36577280, heapUsed=31134692, external=279505, loadavg=[0, 0, 0], uptime=2054.7432857, trace=[], stack=[Error: SQLITE_ERROR: near "categoryId": syntax error]
2018-05-19T14:09:53.271Z - error: uncaughtException: SQLITE_ERROR: near "categoryId": syntax error date=Sat May 19 2018 16:09:53 GMT+0200 (Paris, Madrid (heure d’été)), pid=2404, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=45629440, heapTotal=33697792, heapUsed=16650756, external=358612, loadavg=[0, 0, 0], uptime=2056.8934087, trace=[], stack=[Error: SQLITE_ERROR: near "categoryId": syntax error]
2018-05-19T14:15:34.358Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:15:34 GMT+0200 (Paris, Madrid (heure d’été)), pid=8036, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=54665216, heapTotal=37101568, heapUsed=31482212, external=205896, loadavg=[0, 0, 0], uptime=2397.9799177, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-05-19T14:18:39.239Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:18:39 GMT+0200 (Paris, Madrid (heure d’été)), pid=6488, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=56586240, heapTotal=38903808, heapUsed=33857732, external=362605, loadavg=[0, 0, 0], uptime=2582.8624924, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-05-19T14:22:12.065Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:22:12 GMT+0200 (Paris, Madrid (heure d’été)), pid=1180, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=73252864, heapTotal=54214656, heapUsed=37553572, external=2148478, loadavg=[0, 0, 0], uptime=2795.6886654, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-05-19T14:22:50.384Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:22:50 GMT+0200 (Paris, Madrid (heure d’été)), pid=1684, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=72638464, heapTotal=53583872, heapUsed=40130128, external=2524700, loadavg=[0, 0, 0], uptime=2834.006857, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-05-19T14:24:09.005Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:24:08 GMT+0200 (Paris, Madrid (heure d’été)), pid=1164, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=54394880, heapTotal=36577280, heapUsed=31422324, external=304973, loadavg=[0, 0, 0], uptime=2912.6275392, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-05-19T14:33:01.898Z - error: uncaughtException: SQLITE_ERROR: near "WHERE": syntax error date=Sat May 19 2018 16:33:01 GMT+0200 (Paris, Madrid (heure d’été)), pid=1180, uid=null, gid=null, cwd=C:\Users\PC\Desktop\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.11.1, argv=[C:\Program Files\nodejs\node.exe, C:\Users\PC\Desktop\stock_api\index.js], rss=53833728, heapTotal=36052992, heapUsed=30950380, external=254882, loadavg=[0, 0, 0], uptime=3445.5190188, trace=[], stack=[Error: SQLITE_ERROR: near "WHERE": syntax error]
2018-06-16T22:50:29.719Z - error: uncaughtException: ENOENT: no such file or directory, scandir 'C:\Users\Nidhal\AppData\Local\Temp\jsreport\autocleanup' date=Sun Jun 17 2018 00:50:29 GMT+0200 (Romance Daylight Time), pid=2340, uid=null, gid=null, cwd=E:\stock_api, execPath=C:\Program Files\nodejs\node.exe, version=v8.9.3, argv=[C:\Program Files\nodejs\node.exe, E:\stock_api\index.js], rss=58470400, heapTotal=49340416, heapUsed=44633656, external=295698, loadavg=[0, 0, 0], uptime=2231.3303281, trace=[column=18, file=fs.js, function=Object.fs.readdirSync, line=904, method=readdirSync, native=false, column=20, file=E:\stock_api\node_modules\reap2\index.js, function=null, line=98, method=null, native=false, column=null, file=null, function=Array.forEach, line=null, method=forEach, native=false, column=13, file=E:\stock_api\node_modules\reap2\index.js, function=Reaper.start, line=97, method=start, native=false, column=14, file=E:\stock_api\node_modules\jsreport-core\lib\reporter.js, function=Timeout.setInterval [as _onTimeout], line=667, method=setInterval [as _onTimeout], native=false, column=11, file=timers.js, function=ontimeout, line=475, method=null, native=false, column=5, file=timers.js, function=Timer.unrefdHandle, line=588, method=unrefdHandle, native=false], stack=[Error: ENOENT: no such file or directory, scandir 'C:\Users\Nidhal\AppData\Local\Temp\jsreport\autocleanup',     at Object.fs.readdirSync (fs.js:904:18),     at E:\stock_api\node_modules\reap2\index.js:98:20,     at Array.forEach (<anonymous>),     at Reaper.start (E:\stock_api\node_modules\reap2\index.js:97:13),     at Timeout.setInterval [as _onTimeout] (E:\stock_api\node_modules\jsreport-core\lib\reporter.js:667:14),     at ontimeout (timers.js:475:11),     at Timer.unrefdHandle (timers.js:588:5)]
